frontend/containers/open-call-application-modal/component.tsx

```diff
@@ -0,0 +1,210 @@
+
+  const { data: { data: projects } = { data: [] }, isLoading: isLoadingProjects } =
+    useAccountProjectsList();
+
+  const sortedProjects = useMemo(
+    () => projects.sort((a, b) => a.name.localeCompare(b.name)),
+    [projects]
+  );
+
+  const defaultValues = {
+    project_id: null,
+    message: '',
+  };
+
+  const {
+    control,
+    formState: { errors },
+    clearErrors,
+    handleSubmit,
+    register,
+    reset,
+  } = useForm<OpenCallApplicationForm>({
+    resolver,
+    shouldFocusError: true,
+    shouldUseNativeValidation: true,
+    reValidateMode: 'onChange',
+    defaultValues,
+  });
+
+  const closeModal = () => {
+    applyToOpenCall.reset();
+    reset(defaultValues);
+    clearErrors();
+    onClose();
+  };
+
+  const onSubmit = (values: OpenCallApplicationForm) => {
+    const { project_id, message } = values;
+
+    setIsSubmitting(true);
+
+    applyToOpenCall.mutate(
+      { open_call_id: openCallId, project_id, message },
+      {
+        onSuccess: closeModal,
+        onSettled: () => {
+          setIsSubmitting(false);
+        },
+      }
+    );
+  };
+
+  return (
+    <Modal
+      onDismiss={closeModal}
+      title={formatMessage({ defaultMessage: 'Apply to Open Call', id: 'vC2u0N' })}
+      open={isOpen}
+      size="narrow"
+      scrollable={false}
+    >
+      <form className="flex flex-col" noValidate autoComplete="off">
+        <p className="mb-2 font-serif text-3xl text-green-dark">
+          <FormattedMessage defaultMessage="Apply" id="EWw/tK" />
+        </p>
+
+        <p className="mb-4">
+          <FormattedMessage
+            defaultMessage="Select one of your projects and tell the investor why your project should receive the funding to apply."
+            id="iosznt"
+          />
+        </p>
+        {isLoadingProjects ? (
+          <div className="relative flex items-center justify-center h-20">
+            <Loading visible={true} iconClassName="w-8 h-8" />
+          </div>
+        ) : (
+          <>
+            <div>
+              <Label className="mr-2" htmlFor="project-id">
+                <FormattedMessage defaultMessage="Select project to apply" id="3BlzQw" />
+              </Label>
+              <FieldInfo
+                content={formatMessage({
+                  defaultMessage: 'Choose one of your projects to apply to the open call.',
+                  id: 'b6bipu',
+                })}
+              />
+            </div>
+            <Combobox
+              control={control}
+              controlOptions={{ disabled: false }}
+              aria-required
+              name="project_id"
+              id="project-id"
+              className="mt-2.5 w-full h-10 border border-beige rounded-lg px-4"
+              height="small"
+              placeholder={formatMessage({
+                defaultMessage: 'select project',
+                id: 'kRnyPA',
+              })}
+              aria-describedby="project-id-error"
```

Comment:
I would also link the text that is below the input:
```suggestion
              aria-describedby="project-id-error project-id-note"
```